### ISSUE TITLE
Command-Line - Portable Copy Creation: Add switch to define whether the configuration for the current user should be copied to the new portable version of NVDA

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#11638]
### Is your feature request related to a problem? Please describe.

If using either the "--create-portable-silent" or "--create-portable" command-line switches, a portable copy of NVDA is created; however, unlike when creating a portable version via the GUI, it's not possible to specify whether the configuration data of the current user should be included as part of the creation process. Could we possibly consider adding this? This would be useful for end-users or system admins who are looking to deploy NVDA to multiple machines, but who may wish to use a customised configuration set - or who are distributing NVDA on removable media.
